asyncapi.yaml

```diff
@@ -0,0 +1,351 @@
+#From https://github.com/openapi-contrib/style-guides/pull/4/files
+# These rules will be helping the AsyncAPI people write be better quality, but 
+# have no thoughts on the actual contents of the API
+
```

Comment:
Could we get rid of these empty lines?
It's likely just me, but I find them distracting and they kind of make the processing of document more difficult. :sweat_smile:

Reply:
@P0lip That file was only a todolist to me. I dropped it.